wordpress-mobile/WordPress-Android
# Jetpack Section: Optimize My Site Menu Items

Parent #13629

This PR updates the following for the `My Site` screen menu items:
- The `Activity` string to `Activity log`.
- The `Activity` icon to `ic_gridicons_clipboard_white_24dp`.
- The `Backup` icon to `ic_gridicons_cloud_upload_white_24dp`.
- The `Scan` icon to `ic_baseline_security_white_24dp`.

Light Theme | Dark Theme
-------|------
<img width="250" height="500" alt="before" src="https://user-images.githubusercontent.com/9729923/106918916-50f15e80-6712-11eb-999a-ced36ab0baf5.png"> | <img width="250" height="500" alt="after" src="https://user-images.githubusercontent.com/9729923/106918799-35865380-6712-11eb-8fa0-2634cc6ac7d7.png">

To test:
- Launch the app and go to `My Site` tab.
- Make sure that the above mentioned `string` and `icon` changes are as described (for both the `Light` and `Dark` theme).
- Go to `My Site` -> `Toolbar Avatar` -> `App Settings` -> `Test feature configuration`.
- Enable `MySiteImprovementsFeatureConfig`, scroll down and click the `RESTART THE APP` button.
- Make sure that the above mentioned `string` and `icon` changes are as described (for both the `Light` and `Dark` theme).

PR submission checklist:
